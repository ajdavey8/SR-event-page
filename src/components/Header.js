import React, { Component } from 'react';
import logo from '../images/logo.png';
import '../css/header.css'

export default class Header extends Component {
    render() {
        return (
            <div className="Header row">
                <img src={logo} alt="SpareRoom Logo" />
                <div className="top-nav-bar row">
                    <a href="#"><b>Home</b></a>
                    <a href="#"><b>Events</b></a>
                </div>
                <div className="bottom-nav-bar">
                    <a href="#">Event</a>
                </div>
            </div>
        )
    }
}