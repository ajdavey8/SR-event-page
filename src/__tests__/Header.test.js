import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Header from '../components/header';

describe(Header, () => {
    const wrapper = shallow(<Header/>);
    // it('renders the Header and matches the snapshot', () => {
    //     const tree = renderer
    //     .create(<Header/>)
    //     .toJSON();
    // expect(tree).toMatchSnapshot();
    // })

    it('renders the links in the navigation bar', () => {
        const navlinks = wrapper.find('a').length;
        expect(navlinks).toEqual(3);
    });

})