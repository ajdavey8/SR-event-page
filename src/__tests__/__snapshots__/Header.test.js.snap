// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders the Header and matches the snapshot 1`] = `
<div>
  <img
    alt="SpareRoom Logo"
    src="../images/logo.pngo"
  />
</div>
`;
